src/org/traccar/api/BaseResource.java

```diff
@@ -30,4 +34,8 @@ protected long getUserId() {
         return 0;
     }
 
+    protected String getAcceptHeader() {
+        return headers.getRequestHeaders().getFirst("Accept");
```

Comment:
You should be using `Consumes` annotation instead of manually accessing headers.